crates/sp-consensus-subspace/src/lib.rs

```diff
@@ -84,14 +84,14 @@ impl From<&FarmerPublicKey> for PublicKey {
 }
 
 /// The `ConsensusEngineId` of Subspace.
-const SUBSPACE_ENGINE_ID: ConsensusEngineId = *b"SUB_";
+pub const SUBSPACE_ENGINE_ID: ConsensusEngineId = *b"SUB_";
```

Comment:
Do you really need to make this and `ConsensusLog` public? I intentionally made them private to hide behind nice API so you don't have to deal with inner details explicitly, just use `CompatibleDigestItem`'s API instead.